### PR TITLE
Remove laravelcollective/html from installer, updater and importer

### LINKED ISSUE #1806: Remove laravelcollective/html from installer/updater
After the removal of laravelcollective/html on #1753, there is the need to remove from the installer and updater too.

### PR DESCRIPTION
**Pull request by ArthurHetem**
Continued work of #1753, removing the form helper from installer, updater and importer.

Closes #1806

### CODE REVIEW

File: resources/views/system/importer/step1-configure.blade.php [hunk 1]
```diff
@@ -70,14 +71,14 @@
...
-            {{ Form::input('text', 'db_pass', null, ['class' => 'form-control']) }}
+            <input type="text" name="db_pass" class="form-control" />
           </div>
         </td>
       </tr>
 
       <tr>
         <td colspan="2" style="text-align: right;">
-          {{ Form::submit('Test Database Credentials', ['class' => 'btn btn-info', 'id' => 'dbtest_button']) }}
+          <button type="button" class="btn btn-info" id="dbtest_button">Test Database Credentials</button>
```
Comment (arthurpar06) [inline resources/views/system/importer/step1-configure.blade.php:81]: Shouldn't it rather be `type="submit"` ?

File: resources/views/system/installer/install/steps/step2-db.blade.php [hunk 2]
```diff
@@ -84,14 +89,14 @@
...
-            {{ Form::input('text', 'db_pass', null, ['class' => 'form-control']) }}
+            <input type="password" name="db_pass" class="form-control" />
           </div>
         </td>
       </tr>
 
       <tr>
         <td colspan="2" style="text-align: right;">
-          {{ Form::submit('Test Database Credentials', ['class' => 'btn btn-info', 'id' => 'dbtest_button']) }}
+          <button type="button" class="btn btn-info" id="dbtest_button">Test Database Credentials</button>
```
Comment (arthurpar06) [inline resources/views/system/installer/install/steps/step2-db.blade.php:99]: Here too, shouldn't it be `type="submit"` ?

File: resources/views/system/installer/install/steps/step3-user.blade.php [hunk 3]
```diff
@@ -42,7 +43,10 @@
             <td><p>Airline Country</p></td>
             <td>
               <div class="form-group">
-                {{ Form::select('airline_country', $countries, null, ['class' => 'form-control select2' ]) }}
+                <select name="airline_country" class="form-control select2">
+                  @foreach ($countries as $country)
```
Comment (arthurpar06) [inline resources/views/system/installer/install/steps/step3-user.blade.php:47]: I think there's a missing `</select>`

File: resources/views/system/installer/install/steps/step3a-completed.blade.php [hunk 4]
```diff
@@ -3,18 +3,17 @@
...
 @section('content')
   <div style="align-content: center;">
-    {{ Form::open(['route' => 'installer.complete', 'method' => 'GET']) }}
+    <form method="get" action="{{ route('installer.complete') }}">
+      @csrf
 
     <h4>Installer Completed!</h4>
 
-    <p>Edit the <span class="code">.env</span> file to fill in some additional settings.</p>
+    <p>Edit the <span class="code">env.php</span> to fill in some additional settings.</p>
```
Comment (arthurpar06) [inline resources/views/system/installer/install/steps/step3a-completed.blade.php:11]: We are using `.env` instead of `env.php` since #1597

**Review by arthurpar06 — CHANGES_REQUESTED** [review]

File: resources/views/system/importer/step1-configure.blade.php (hunk 1 above)
Comment (ArthurHetem) [reply resources/views/system/importer/step1-configure.blade.php:81]: This one is used in js functions to test the database credentials, not to submit the form

Comment (ArthurHetem) [reply resources/views/system/importer/step1-configure.blade.php]: Now i saw that there is a preventdefault, fixed on the latest commit

**Review by arthurpar06 — APPROVED** [review]

**Pull request merged**